src/lib.rs

```diff
@@ -514,6 +516,9 @@ struct SyncClient {
     tcp_connect_timeout: Duration,
     delay_generator: DelayGenerator,
     subscriptions: HashMap<Sid, Subscription>,
+    request_inbox_mapping: HashMap<String, MpscSender<Msg>>,
+    request_wildcard_subscription: Option<Sid>,
```

Comment:
This probably doesn't need to be an Sid. We really only use this to ensure the request handler exists.